apps/web/screens/ApiCatalogue/ApiCatalogue.tsx

```diff
@@ -200,19 +204,96 @@ const ApiCatalogue: Screen<ApiCatalogueProps> = ({
     },
   ]
 
+  const navigationItems = [
```

Comment:
These items are defined in two places. Shouldn't they come from a better source of truth?

Reply:
I am not quite sure what you mean, can you elaborate a little more?  Do you mean, these strings should not come from contentful?